### DISCUSSION TITLE
multiple FS for cephfs

### DISCUSSION THREAD

**Discussion by lumir-sliva** [ceph/ceph-csi#3804] [General]
It would be nice for ceph-csi to be able to choose from multiple FS for the csi-config-map, we have clusters where we want to use multiple FS, because of the seperate client I/O. If you have 1 unresponsive client it can affect all the clients, this would not be the case with multiFS. 

Could you consider this feature request?

**Comment by Rakshith-R** [comment]
> It would be nice for ceph-csi to be able to choose from multiple FS for the csi-config-map, we have clusters where we want to use multiple FS, because of the seperate client I/O. If you have 1 unresponsive client it can affect all the clients, this would not be the case with multiFS.
> 
> Could you consider this feature request?

@lumir-sliva, filesystem is a storageclass parameter https://github.com/ceph/ceph-csi/blob/devel/examples/cephfs/storageclass.yaml#L19 .

You can create multiple storageclasses with unique FS and PVC from them.

**Comment by Madhu-1** [comment]
@lumir-sliva you can create multiple storageclass if you want to use multiple filesystems.

**Comment by lumir-sliva** [comment]
thank you, I will try it.
